### PR TITLE
Fixes steel lump recipe anomaly

### PR DESCRIPTION
**Pull request by DangerNoodle**
Simple fix for recipe to produce lumps of steel. It should accept chunks of steel or scrap metal, not both.

### CODE REVIEW

File: data/json/recipes/other/materials.json [hunk 1]
```diff
@@ -32,8 +32,7 @@
       [ [ "forge", 80 ], [ "char_forge", 80 ] ]
     ],
     "components": [
```
Comment (mugling) [inline data/json/recipes/other/materials.json]: All on one line with `components`?

Comment (DangerNoodle) [inline data/json/recipes/other/materials.json]: I recall @Coolthulhu stating that `electronics/parts.json` is the current recommended reference for styling, and it is formatted in this way there. It does not give a reference for when you have a choice of components to use, but it does follow the same conventions for when multiple tools are permitted.

Comment (mugling) [inline data/json/recipes/other/materials.json]: You can run the JSON linter on a file via `./tools none data/json/foo.json` to get canonical format

Comment (DangerNoodle) [inline data/json/recipes/other/materials.json]: Not sure where that is, have never tried.

Comment (mugling) [inline data/json/recipes/other/materials.json]: Give it a try then

Comment (DangerNoodle) [inline data/json/recipes/other/materials.json]: Will have to later on when I get the chance, and might clean up a few assorted recipe files, if I can determine that they are not currently the subject of a pull request. I suspect that the risk of multiple merge conflicts is the reason there has not been a large-scale cleanup of recipe formatting.

Comment (mugling) [inline data/json/recipes/other/materials.json]: Attempts at cleanup are very welcome. If you proceed one file at a time then the risk of merge conflict is acceptably low.

For recipes you can sort via:
`./tools/cleanup.sh recipe data/json/foo.json`

For items you can sort via:
`./tools/cleanup.sh item data/json/foo.json`

Keep linting as a separate commit to any other (preferably minimal) changes you make

Comment (Coolthulhu) [inline data/json/recipes/other/materials.json]: Linting requires a Perl installation with JSON module, it's not trivial to run on Windows.
Hence why I usually recommend a file that has proper formatting - it's easier than giving a whole tutorial on how to set up Perl.

Comment (mugling) [inline data/json/recipes/other/materials.json]: I do believe there is a perl packing tool (packs an interpreter, module and script) as an exe. It might be worth distributing such a binary with the project?

Comment (DangerNoodle) [inline data/json/recipes/other/materials.json]: That almost sounds more tedious and complicated that fixing the formatting personally, if it can't simply be done by running the tool through command line.

Comment (mugling) [inline data/json/recipes/other/materials.json]: If you're using an *nix system then go with the commands I posted. I would suspect most windows users would prefer double clicking a binary and waiting a few moments as opposed to manually checking a file?
